pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/DirectProxyHandler.java

```diff
@@ -71,11 +66,11 @@
 
     @Getter
     private final Channel inboundChannel;
+    private final ProxyConnection proxyConnection;
     @Getter
     Channel outboundChannel;
     @Getter
     private final Rate inboundChannelRequestsRate;
-    protected static Map<ChannelId, ChannelId> inboundOutboundChannelMap = new ConcurrentHashMap<>();
```

Comment:
great to see this removed!

Reply:
Great remove.

Look like the data in this map will never remove.